kornia/augmentation/container/augmentation_sequential.py

```diff
@@ -0,0 +1,268 @@
+from typing import Tuple, Union, Optional, List, Dict, cast
+
+import torch
+import torch.nn as nn
+
+from kornia.geometry import transform_points, transform_boxes
+from kornia.augmentation.base import _AugmentationBase, IntensityAugmentationBase2D, GeometricAugmentationBase2D
+from kornia.constants import DataCategory
+from .sequential import Sequential
```

Comment:
I would avoid confusion naming with `torch.nn.Sequential`

Reply:
what do you have in mind?